### PR TITLE
Format PHP language specific partial

### PR DESCRIPTION
**Pull request by gaelreyrol**
Some formatting and cleaning on the PHP language specific partial file !

### CODE REVIEW

File: partials/language-specific-deploy/php.md [hunk 1]
```diff
@@ -36,15 +36,16 @@ of the configuration file *clevercloud/php.json* with the absolute path (*from t
...
-      "webroot": "/public"
+        "webroot": "/public"
     }
-  }
+}
 ```
+
 {{< alert "warning" "Warning:" >}}
-    <p>Please note the absolute path style: `/public`. The change of the webroot will be rejected during the deployment if the target directory does not exist or is not a directory.</p>
+    Please note the absolute path style: `/public`. The change of the webroot will be rejected during the deployment if the target directory does not exist or is not a directory.
```
Comment (BlackYoup) [inline partials/language-specific-deploy/php.md:48]: The backticks here won't be rendered

File: partials/language-specific-deploy/php.md [hunk 2]
```diff
@@ -432,10 +426,9 @@ To enable this feature, you need to:
  - create an [environment variable](#setting-up-environment-variables-on-clever-cloud) named `SESSION_TYPE` with the value `redis`.
 
 {{< alert "warning" "Warning:" >}}
-    <p>You must have a [Redis]({{< ref "deploy/addon/redis.md" >}}) add-on
-    [linked with your application](#linking-a-database-or-any-other-add-on-to-your-application)
-    to enable PHP session storage in Redis.<br />
-    If no Redis add-on is linked with your application, the deployment will fail.</p>
+    You must have a [Redis]({{< ref "deploy/addon/redis.md" >}}) add-on [linked with your application](#linking-a-database-or-any-other-add-on-to-your-application) to enable PHP session storage in Redis.
```
Comment (BlackYoup) [inline partials/language-specific-deploy/php.md:429]: Same here for all the markdown stuff

Comment (BlackYoup) [reply partials/language-specific-deploy/php.md:429]: There is also another warning that isn't in this diff that starts with "Environment variables are displayed" that contains markdown tags inside the HTML, could you also fix it?

**Review by BlackYoup — CHANGES_REQUESTED** [review]
A few changes to make regarding markdown inside alerts but other than that's, LGTM

File: partials/language-specific-deploy/php.md (hunk 1 above)
Comment (gaelreyrol) [reply partials/language-specific-deploy/php.md:48]: On my side it is rendered, I use `hugo v0.83.1`.

File: partials/language-specific-deploy/php.md (hunk 2 above)
Comment (gaelreyrol) [reply partials/language-specific-deploy/php.md:429]: Same thing here and I don't get this warning, what command do you run ?

File: partials/language-specific-deploy/php.md (hunk 1 above)
Comment (BlackYoup) [reply partials/language-specific-deploy/php.md:48]: Mh, that's not the version we currently use but we can update. Did you notice any issues with this version?

Comment (BlackYoup) [reply partials/language-specific-deploy/php.md:48]: I'll upgrade the hugo version, it's indeed okay with the last one

**Pull request merged**